app/components/select/birth-date.js

```diff
@@ -34,5 +37,14 @@ export default Component.extend({
       }
       return results;
     }).apply(this);
-  }.property()
+  }.property(),
+
+  update(property, value) {
+    set(this, property, value);
+    let { selectedDay, selectedMonth, selectedYear }
+      = getProperties(this, 'selectedDay', 'selectedMonth', 'selectedYear');
+
+    let onChange = get(this, 'onChange');
+    onChange(selectedDay, selectedMonth, selectedYear);
+  }
```

Comment:
This single action get's called for all 3 selection changes. It allows us to pass out a single `onChanged` to the parent, with all 3 values. Makes the interface with the component simpler than it would be if we sent out each action separately.